vmaas/rest/schemas.py

```diff
@@ -60,6 +61,12 @@
 
 _updates_top = {'update_list': {str: dict}}
 _updates_top_repolist = {'repository_list': [str], 'update_list': {str: dict}}
+_updates_top_basearch = {'basearch': str, 'update_list': {str: dict}}
+
+if GH(241).blocks:
```

Comment:
Guys, if we are going to be calling to GH from the codebase, we need to have an API token set up for authenticated interaction. Otherwise, we will be hitting rate limits on the CI (we just have). I'll be adding a token to the internal repo and loading it so that we don't hit the limit anymore. Just FYI.

@mkoura @psegedy